DEVELOPMENT.md

```diff
@@ -90,3 +90,25 @@ pnpm -- wp-env destroy
 Each of the [plugins in our repository](plugins) support using this tool to spin up a development environment. Note that rather than having a single top-level environment, each plugin has its own. This is done in order to prevent conflicts between them.
 
 Please check out [the official documentation](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/) if you would like to learn more about this tool.
+
+## Troubleshooting
+
+### Installing PHP in Unix (e.g. Ubuntu)
+
+Many unix systems such as Ubuntu will have PHP already installed. Sometimes without the extra packages you need to run Wordpress and this will cause you to run into troubles.
```

Comment:
```suggestion
Many unix systems such as Ubuntu will have PHP already installed. Sometimes without the extra packages you need to run WordPress and this will cause you to run into troubles.
```